go/tao/keys.go

```diff
@@ -802,6 +802,14 @@ type Keys struct {
 	Cert         *x509.Certificate
 }
 
+const (
+	X509Path           = "cert"
+	PBEKeysetPath      = "keys"
+	PBESignerPath      = "signer"
+	SealedKeysetPath   = "sealed_keyset"
+	UnsealedKeysetPath = "unsealed_keyset"
```

Comment:
This name makes it sound like these keys have been unsealed, when in fact they are just plaintext. How about PlaintextKeysetPath, to make that clear?